assets/prototype/domain/shared/entities/tickets/predicates/filters/percentSoldAtOrAbove/index.ts

```diff
@@ -13,25 +13,29 @@ type PercentSoldAtOrAboveProps = {
 	percentage: number;
 	tickets: Ticket[];
 };
+type FilterFnProps = {
+	percentage: number;
+	ticket: Ticket;
+};
 
-/**
- * @param {Object} ticket event ticket object
- * @param {number} percentage
- * @return {boolean} true if sold/qty >= percentage
- */
-const percentSoldAtOrAbove = ({ percentage, tickets }: PercentSoldAtOrAboveProps) => {
-	const calc = (ticket: Ticket) => {
+export const filterFn = ({ percentage, ticket }: FilterFnProps): boolean => {
+	const calc = (ticket: Ticket): boolean => {
 		const { quantity, sold } = ticket;
 		const checkIfSoldAndQtyAreNumbers = is(Number, sold) && is(Number, quantity);
```

Comment:
wonder if we should also verify that quantity is not zero to avoid that error?

Reply:
Done: https://github.com/eventespresso/event-espresso-core/pull/2302/commits/54a97a667a9e3b3ea7c0ee08df3fa91ce62f6052
Thanks